pkg/imgpkg/image/tar_image.go

```diff
@@ -141,12 +151,16 @@ func (i *TarImage) addFileToTar(fullPath, relPath string, info os.FileInfo, tarW
 	if runtime.GOOS == "windows" {
 		relPath = strings.ReplaceAll(relPath, "\\", "/")
 	}
+	filePermission := int64(info.Mode() & 0700)
```

Comment:
Why do we need to make it '7xx' permissions by default exactly ?

Reply:
> I assume by bitwise AND'ing we are making it at least 0700 here

I am not really good at this, but I think we are making it _at most_ 0700, which does make sense (it would be at least 0700 if we were doing a `bitwise OR`)

Reply:
The bitwise here ensures that we only capture the part of the permissions associated with the User permissions. So what this does basically is to remove Group and All Permissions

Reply:
Ahhh yes, the opposite :D Need to bitwise OR my brain this morning to make sure its all on